plugins/net_plugin/net_plugin.cpp

```diff
@@ -2685,9 +2766,6 @@ namespace eosio {
          ( "p2p-server-address", bpo::value<string>(), "An externally accessible host:port for identifying this node. Defaults to p2p-listen-endpoint.")
          ( "p2p-peer-address", bpo::value< vector<string> >()->composing(), "The public endpoint of a peer node to connect to. Use multiple p2p-peer-address options as needed to compose a network.")
          ( "agent-name", bpo::value<string>()->default_value("\"EOS Test Agent\""), "The name supplied to identify this node amongst the peers.")
-#if 0 //disabling block summary support
```

Comment:
Why delete the send-whole-blocks code farther down, but leave this commented out?